app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+        elif date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        elif not visitor.get("wearing_a_mask"):
+            raise NotWearingMaskError("NotWearingMaskError")
+        else:
+            return f"Welcome to {self.name}"
```

Comment:
provide more informatically errors text like "Visitor should be vaccinated" 
use "if" instead of "elif" cause in a case of exceptions raising u will be thrown out of function
remove redundant "else"

Reply:
Ok, I agree. But errors text don't determine in readme. 

![Записати](https://user-images.githubusercontent.com/39659654/230602786-24226908-5013-41fa-a575-103a61d42de3.PNG)